### ISSUE TITLE
Duplicate Hipchat Notifications

### ISSUE THREAD

**Issue by zachsnow** [linovia/sentry-hipchat#30]
I'm getting duplicate messages to HipChat when this plugin is enabled. For instance using Raven on the frontend, simply doing this:

`Raven.captureMessage("Hello!");`

Results in 2 messages in the connected HipChat room, both linking to the same event. Disabling the plugin results in no messages, as expected. I've also tried creating a brand new room and received the same duplicate notifications. So I don't think there's a different integration causing issues.

I assume that I must have something incorrectly configured but I can't seem to sort out what.

**Comment by dcramer** [comment]
@zachsnow what version of Sentry and is this master of sentry-hipchat?

**Comment by zachsnow** [comment]
Hey, it's actually the implementation on getsentry.com; maybe not much more to do then?

**Comment by dcramer** [comment]
@zachsnow are you using any of the rate-limit based rules? We identified a bug last week that would cause duplicates there (unexpectedly, at least)

**Comment by zachsnow** [comment]
We don't have any throttling or rate limiting set up. However, I noticed that there was a rule in the "Rules" area (on getsentry.com) for Hipchat integration that disappeared when I disabled the plugin, and did not reappear when I re-enabled it. No idea why -- I had disabled it and re-enabled it a few times when testing before creating this issue. And of course now I'm not receiving duplicates. Sorry to trouble you!
